### PR TITLE
Add an example for BigInt64Array()

### PR DESCRIPTION
**Pull request by MrBrain295**
### Description
Add an example for BigInt64Array().

### Motivation
There isn't already an example.

### CODE REVIEW

**Review by NiedziolkaMichal — CHANGES_REQUESTED** [review]
Thanks for filing the pull request. I think it would be better if the following changes were made:
- You need to fix the warnings to pass ESLint
- Since it's an example showing the general functionality of `BigInt64Array`, it should show multiple different features
- Each `console.log` should have a `// Expected Output: <output_here>` comment below
- Other examples use the name `uint8`, so I think the most appropriate variable name would be `bigint64` here
- Try to keep the example as concise as possible.

File: live-examples/js-examples/bigint/bigint64.js
Comment (Josh-Cena) [inline live-examples/js-examples/bigint/bigint64.js]:
```suggestion
bigint64[0] = 5886014448488689n;
bigint64[1] = 1881938909131133n;
bigint64[2] = 1898875537769492n;
bigint64[3] = 2156704815705747n;
bigint64[4] = 3337090060496542n;
```

(Number literals already lose precision)

File: live-examples/js-examples/bigint/bigint64.js
Comment (Josh-Cena) [inline live-examples/js-examples/bigint/bigint64.js]:
```suggestion
bigint64[0] = 6118793953620967n;
```

File: live-examples/js-examples/bigint/bigint64.js
Comment (Josh-Cena) [inline live-examples/js-examples/bigint/bigint64.js]: Update the output here; also there needs to be a space after `//`

**Review by Josh-Cena — COMMENTED** [review]
I don't know if the `BigInt()` usage is intentional, but I suggest we show something that doesn't lose precision.

**Comment by MrBrain295** [comment]
@Josh-Cena can you please re-review?

File: live-examples/js-examples/bigint/bigint64.js
Comment (Josh-Cena) [inline live-examples/js-examples/bigint/bigint64.js]:
```suggestion
bigint64.set([100n, 200n], 1);
```

**Review by Josh-Cena — COMMENTED** [review]
LGTM modulo one change. Waiting for @NiedziolkaMichal for the approval

**Comment by MrBrain295** [comment]
@NiedziolkaMichal could you please review?

**Review by NiedziolkaMichal — DISMISSED** [review]
- You need to fix the warnings to pass ESLint
- There are 4 scrollbars because of too long output:
image
- We always place `// Expected Output` in the line under `console.log` and we typically place an empty line under it
- Try to keep the example as concise as possible

**Comment by MrBrain295** [comment]
How do I fix the 'BigInt64Array' is not defined error

**Comment by NiedziolkaMichal** [comment]
It needs to be added to `globals` in [.eslintrc](https://github.com/mdn/interactive-examples/blob/main/.eslintrc)

**Comment by MrBrain295** [comment]
I don’t have much eslint experience so could you please explain how to do that?

**Comment by MrBrain295** [comment]
@NiedziolkaMichal I can't figure out why the test is failing but I did get eslint to work.

**Comment by caugner** [comment]
@NiedziolkaMichal @MrBrain295 I'm very sorry about the failing workflow. https://github.com/mdn/interactive-examples/pull/2429 should resolve it.

**Comment by NiedziolkaMichal** [comment]
@MrBrain295 I have added PR #2439 which should resolve this problem. Could you still take a look into other problems with the example that I mentioned? I have also noticed that you have placed the new example into a new directory called `bigint`. I think it would be better to place it in `typedarray` together with other related examples.

**Comment by queengooborg** [comment]
Thanks @MrBrain295 for your hard work on this -- since this PR has been idling for a while, I made the requested changes!

**Review by queengooborg — DISMISSED** [review]

**Comment by queengooborg** [comment]
Okay, _now_ this should be ready for re-review!

**Review by bsmth — APPROVED** [review]
Thanks all, this looks good to me, merging shortly:

<details>

image

</details>

**Pull request merged**
